src/types/metaFactory.ts

```diff
@@ -117,5 +131,31 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
           }
         }
       })}
+  },
+  candyMachine: {
+    newContractCreatedEvent: '',
+    metadataAccountToNftFactory: (metadataAccount, metaFactory) => {
+      return {
+        id: metadataAccount.mintAddress.toBase58(),
+        contractType: NFTContractTypeName.candyMachine,
+        platformId: metaFactory.platformId,
+        standard: NFTStandard.METAPLEX,
+        name: metadataAccount.name,
+        symbol: metadataAccount.symbol,
+        autoApprove: true, 
+        approved: true, 
+        typeMetadata: {
+          collection: metaFactory.id,
+          ...metaFactory.typeMetadata,
+          overrides: {
+            ...metaFactory.typeMetadata?.overrides,
+            artist: {
+              artistId: candyMachineArtistId(metadataAccount),
```

Comment:
artist id extracted here because we don't store the metadata account to extract from later